ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/metacard-history/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from './metacard-history'
```

Comment:
✏️ I second @bellcc's earlier comment about adding the Codice copyright header to this file.